patches/minecraft/net/minecraft/entity/effect/LightningBoltEntity.java.patch

```diff
@@ -1,5 +1,28 @@
 --- a/net/minecraft/entity/effect/LightningBoltEntity.java
 +++ b/net/minecraft/entity/effect/LightningBoltEntity.java
+@@ -30,6 +_,7 @@
+    private boolean field_184529_d;
+    @Nullable
+    private ServerPlayerEntity field_204810_e;
++   private float damage = 5.0F;
```

Comment:
Does this need to be saved to NBT? I get the feeling most people will call the setter right after the constructor. On the odd chance the bolt lives long enough for the world to close, its damage will be wrong.

Reply:
It appears that LightningBoltEntity is marked with .noSave(), and from my testing, does not save into the world. So the entity will be gone anyway on world restart.